pkg/loki/modules.go

```diff
@@ -494,6 +494,7 @@ func (t *Loki) initQueryFrontend() (_ services.Service, err error) {
 		queryrange.StatsHTTPMiddleware,
 		serverutil.NewPrepopulateMiddleware(),
 		serverutil.ResponseJSONMiddleware(),
+		serverutil.ExtractQueryTagsMiddleware(),
```

Comment:
would be nice if I could lock this middleware chain for Query handler by some tests!. 

Also bit tricky to write one with all the dependency of `initQueryFrontend`.

Any thoughts are welcome!